### DISCUSSION TITLE
SF .NET5: Where have the slickgrid scripts gone?

### DISCUSSION THREAD

**Discussion by JohnRanger** [serenity-is/Serenity#5454] [Help] [answered]
Hi all, hi @VictorTomaili, hi @volkanceylan 

migrating an existing project to a new Serene .NET5, I wanted  - as usual - to include the slick.autosizecolumns.ts script - but I don't find anymore the slickgrid folder and files under wwwroot/scripts which were previously included within a standard serene project (so that I just could add the respective script within the plugins folder of slickgrid.

Where are now all these slickgrid scripts? A search didn't reveal any slick files. A textual search revealed a lot of calls to slickgrid functions from within serenity.corelib.js.

Thanks for shedding some light on this.

With kind regards,

John

**Comment by WasimAhmad** [comment, marked as answer]
https://serenity.is/docs/migration/v5-0-12-to-v5-0-20

**Reply by JohnRanger** [reply]
Hi @WasimAhmad ,

thanks - I have obviously missed the info there regarding the Serenity.Assets part ;-) Thanks for pointing this out to me.

With kind regards,

John

**Reply by WasimAhmad** [reply]
you are welcome
